### ISSUE TITLE
[Bug] Missing MapboxGeoJSONFeature 

### ISSUE THREAD

**Issue by lightstrike** [visgl/react-map-gl#2264]
### Description

When importing `MapboxGeoJSONFeature` like so:

```ts
import type { MapboxGeoJSONFeature, MapLayerMouseEvent, MapRef } from 'react-map-gl';
```

The following error is thrown:

`'"react-map-gl"' has no exported member named 'MapboxGeoJSONFeature'. Did you mean 'MapGeoJSONFeature'?`

This [type exists in the docs](https://visgl.github.io/react-map-gl/docs/api-reference/types#mapboxgeojsonfeature), yet [no where else in the codebase](https://github.com/search?q=repo%3Avisgl%2Freact-map-gl%20MapboxGeoJSONFeature&type=code). I am trying to use a type that includes the `coordinates` property for the following:

```ts
mapRef?.current?.easeTo({
  center: feature.geometry?.coordinates,
  zoom,
  duration: 500
});
```

If I try to replace `MapboxGeoJSONFeature` with `MapGeoJSONFeature`, the following error is thrown:

```
Property 'coordinates' does not exist on type 'Geometry'.
Property 'coordinates' does not exist on type 'GeometryCollection<Geometry>'.
```

### Steps to Reproduce

1. Copy [clusters example code](https://github.com/visgl/react-map-gl/tree/7.1-release/examples/clusters).
2. Replace lines 9-10 in `app.tsx` with `import type { MapboxGeoJSONFeature, MapLayerMouseEvent, MapRef } from 'react-map-gl';`
3. Run `npm start` or view TS import error in code editor.

### Environment

- Framework version:  react-map-gl@7.1.4
- Map library: mapbox-gl@2.15.0
- Browser: Chrome 116.0
- OS: macOS 13.5

**Comment by Pessimistress** [comment]
The type is now called `MapGeoJSONFeature`. There is an open PR that updates the docs.

**Comment by Pessimistress** [comment]
Backward compatibility is fixed in v7.1.5.
